changelogs/CHANGELOG-v1.20.0.md

```diff
@@ -1,7 +1,16 @@
-We are delighted to present the first beta for Contour v1.20.0, our layer 7 HTTP reverse proxy for Kubernetes clusters.
+We are delighted to present version v1.20.0 of Contour, our layer 7 HTTP reverse proxy for Kubernetes clusters.
```

Comment:
note, I deleted the `v1.20.0-beta.1` changelog as part of this, I didn't see a good reason to keep it around, but if anyone feels differently let me know. The contents of it are still contained in [the GitHub release](https://github.com/projectcontour/contour/releases/tag/v1.20.0-beta.1).